docs/04-operation-guides/troubleshooting/eventing/evnt-04-free-jetstream-storage.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Free NATS JetStream file storage when it gets full
+---
+
+## Symptom
+
+Free NATS JetStream file storage when it gets full.
+
+## Cause
+
+NATS JetStream uses [Interest retention policy](https://docs.nats.io/using-nats/developer/develop_jetstream/model_deep_dive) in Kyma per default.
+It means, that as long as there are consumers on the stream, which the published event's subject, the messages will be kept in the stream if they cannot be delivered to the sink.
+
+In some cases, it might happen, that the NATS JetStream storage gets full due to too many undelivered events.
```

Comment:
@NHingerl Just changed a word. What do you think?

```suggestion
If there are too many undelivered events, the NATS JetStream storage may get full. To prevent event loss, the backend stops receiving events, and no further events can be persisted in the stream.
```

Reply:
I took the @NHingerl idea, would that be ok @raypinto ?

Reply:
I think that messages are persisted **in** the stream and not **to** the stream. But @NHingerl could suggest better :)

Reply:
That seems like a technical question rather than linguistic, I'll go with Ray's suggestion. I may have misunderstood where/how the persisting happens.